.changeset/swift-kids-help.md

```diff
@@ -0,0 +1,83 @@
+---
+"amplify-ui-angular-mono": patch
+"@aws-amplify/ui-react": minor
+"@aws-amplify/ui-angular": patch
+---
+
+Angular: Add `amplify-dialcodeselect` class which contains the previous countrycodeselect styles
+Vue: Add `amplify-dialcodeselect` class which contains the previous countrycodeselect styles
+React: Added 'dialCode' versions of all 'countryCode' props so that users can begin migrating away from the deprecated `countryCode`.
+
+```
+// Added dialCode props
```

Comment:
Instead of including all this code in the changelog, perhaps we can just outline which props have been changed? For example:

`countryCodeLabel` => `dialCodeLabel`
`countryCodeName` => `dialCodeName`
etc